client/src/app/modals/deploy-diagram/validation-errors/getNetworkErrorMessage.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Do we intend to reuse this file? Otherwise I'd rather inline this stuff where it is used.